internals/search.py

```diff
@@ -208,7 +208,7 @@ def parse_query_value_list(
 # Structured terms look like: FIELD OPERATOR VALUE.
 # Full-text terms look like: SINGLE_WORD, or like: "QUOTED STRING".
 TERM_RE = re.compile(
-    '(?P<logical>%s)?(?:(?P<field>%s)(?P<op>%s)(?P<val>%s)|(?P<textterm>%s))\s+' % (
+    r'(?P<logical>%s)?(?:(?P<field>%s)(?P<op>%s)(?P<val>%s)|(?P<textterm>%s))\s+' % (
```

Comment:
Just checking: did you mean to change this line?

Reply:
Yes.  Thanks for pointing that out.  I just added it to the PR description.

Reply:
I just checked and the other patterns above in the file are also raw strings so this makes sense.